go-controller/hybrid-overlay/pkg/controller/node_linux.go

```diff
@@ -735,15 +732,17 @@ func (n *NodeController) syncFlows() {
 			cookie = "0" + cookie
 		}
 		if cacheEntry, ok := n.flowCache[cookie]; ok {
-			// we ignore certain cookies for learning to avoid a case where a NS was updated with a new vtep
-			// and we accidentally pick up the old vtep flow and cache it. This should only ever happen on a pod update
-			// with an NS annotation VTEP change. We only need to ignore it for one iteration of sync.
-			if cacheEntry.ignoreLearn {
```

Comment:
we still need this logic, ignoreLearn is still going to need to be used, its just we can always ignore learn when we do a pod add or update